libs/perun/pipes/src/lib/select-application-link.pipe.ts

```diff
@@ -0,0 +1,52 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import { Application, Group, Member } from '@perun-web-apps/perun/openapi';
+
+@Pipe({
+  name: 'selectApplicationLink',
+})
+export class SelectApplicationLinkPipe implements PipeTransform {
+  /**
+   * Select the appropriate url for the given application (for Vo, Group or Member)
+   *
+   * @param application the given application
+   * @param disableRouting returns null if routing is disabled
+   * @param group group if it os a group application
+   * @param member member if it should be a link for a member application
+   */
+  transform(
+    application: Application,
+    disableRouting: boolean,
+    group: Group,
+    member: Member
+  ): string[] {
+    if (!disableRouting) {
```

Comment:
Please return early if routing is disabled so there is no nesting and hopefully help readability.